app/src/main/java/com/sunasterisk/musixmatch/utils/PlaylistUtils.java

```diff
@@ -0,0 +1,64 @@
+package com.sunasterisk.musixmatch.utils;
+
+import android.content.ContentResolver;
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.net.Uri;
+import android.provider.MediaStore;
+
+import com.sunasterisk.musixmatch.data.model.Track;
+
+/**
+ * Created by superme198 on 14,April,2019
+ * in Music_Match__1.
+ */
+public class PlaylistUtils {
+
+    private Context mContext;
+    private static ContentResolver mContentResolver;
+    private static Cursor mCursor;
+    private static PlaylistUtils sInstance;
+    private static final String VOLUME_NAME = "external";
+    private static final String URI_STRING = "content://media";
+    private static final int INCREASE_INDEX = 1;
+    private static final int BASE_INDEX = 0;
+    private static final int ERROR_INDEX = -1;
+
+    private PlaylistUtils(Context context) {
+        mContext = context;
+        mContentResolver = mContext.getContentResolver();
+    }
+
+    public static PlaylistUtils getInstance(Context context) {
```

Comment:
Utils class toàn phuơng thức static thì e cần singleton làm gì?

Reply:
trong trường hợp này em nên đặt tên class khác đi
và các method k nên để là static nữa